gulpfile.js

```diff
@@ -94,15 +94,15 @@ var appBundleBaseConfig = {
 };
 
 var appBundles = [{
-  // The sidebar application for displaying and editing annotations
+  // The sidebar application for displaying and editing annotations.
   name: 'app',
   transforms: ['coffee'],
-  entry: './src/scripts/app',
+  entry: './src/sidebar/app',
 },{
-  // The Annotator library which provides annotation controls on
-  // the page and sets up the sidebar
+  // The annotation layer which handles displaying highlights, presenting
+  // annotation tools on the page and instantiating the sidebar application.
   name: 'injector',
```

Comment:
The name "injector" doesn't really make sense any more but it cannot be changed because the service still hardcodes the name of the bundle.